api/adobe_vendor_id.py

```diff
@@ -1009,7 +1010,7 @@ def create_function():
         return patron_identifier_credential, delegated_identifier
 
 
-class ShortClientTokenLibraryConfigurationScript(Script):
+class VendorIDLibraryConfigurationScript(Script):
```

Comment:
I renamed this because "Vendor ID" is the term we use for NYPL's library configuration (which includes Short Client Token secrets for many other libraries) and "Short Client Token" is the term we use when helping other libraries get their configuration set up.